pkg/artifactory/resource_artifactory_permission_targets_v1.go

```diff
@@ -113,6 +113,11 @@ func resourcePermissionTargetV1Read(d *schema.ResourceData, m interface{}) error
 	c := m.(*ArtClient).ArtOld
 
 	permissionTarget, resp, err := c.V1.Security.GetPermissionTargets(context.Background(), d.Id())
+
+	if resp == nil {
```

Comment:
Check `err` first. If you check error first you probably never need to check if `resp` is `nil`.